api/portainer.go

```diff
@@ -1071,6 +1078,10 @@ const (
 	AzureRegistry
 	// CustomRegistry represents a custom registry
 	CustomRegistry
+	// DockerhubRegistry represents a dockerhub registry
```

Comment:
I believe this should be removed from this PR ? And the associated value check in the registry creation operation should be updated to match `4` instead of `5`.

Reply:
Was developping both features (this and dockerhub), and as we are using iota to define the types I was forced to do so. Need to ensure we have the same number for this type on frontend also.
And we need to change dockerhub codebase (or at least ping it) to ensure we don't have the same number for both types.

Reply:
Yeah IMO both implementations should be done separately.